src/System.Net.Mail/src/System/Net/Mail/SmtpConnection.cs

```diff
@@ -40,6 +40,9 @@ internal partial class SmtpConnection
         private readonly ChannelBinding _channelBindingToken = null;
         private bool _enableSsl;
         private X509CertificateCollection _clientCertificates;
+#pragma warning disable CS0414
+        private bool _aborted; // Tracks whether Abort was called, for debugging https://github.com/dotnet/corefx/issues/40711.
```

Comment:
Do I need to worry about the JIT removing this?

Reply:
No.

But I'm wondering why this needs to be checked in.  Are you expecting to see it in a dump someone else provides?

Reply:
> But I'm wondering why this needs to be checked in. Are you expecting to see it in a dump someone else provides?

yes, from a CI coredump. This is what I understood from the previous dump: https://github.com/dotnet/corefx/issues/40711#issuecomment-551816472.

Reply:
(Though, I'd need to check, it's possible the linker will remove it. You can check that by looking at the IL of the compiled .dll locally.)

Reply:
I had a look at ./runtime/netcoreapp-Linux-Release-x64/System.Net.Mail.dll, and the code is there:
```
	.method assembly hidebysig 
		instance void Abort () cil managed 
	{
		// Method begins at RVA 0xb5b8
		// Code size 135 (0x87)
		.maxstack 3
		.locals (
			[0] class System.Net.Mail.SmtpConnection,
			[1] bool
		)

		IL_0000: ldarg.0
		IL_0001: ldc.i4.1
		IL_0002: stfld bool System.Net.Mail.SmtpConnection::_aborted
```